src/System.Reflection.Metadata/src/System/Reflection/Metadata/PortablePdbReaderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Reflection.PortablePdb;
+
+namespace System.Reflection.Metadata
+{
+    // EditorBrowsable(Never) so that we don't clutter completion list with this type because a user that only has System.Reflection.Metadata 
+    // imported and has type PE is likely looking to resolve PEReader from the System.Reflection.PortableExecutable and not looking to invoke
+    // these extensions as regular statics.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class PortablePdbReaderExtensions
+    {
+        /// <summary>
+        /// Gets a <see cref="MetadataReader"/> from a <see cref="PortablePdbReader"/>.
+        /// </summary>
+        /// <remarks>
+        /// The caller must keep the <see cref="PortablePdbReader"/> alive and undisposed throughout the lifetime of the metadata reader.
+        /// </remarks>
+        public static unsafe MetadataReader GetMetadataReader(this PortablePdbReader reader, MetadataReaderOptions options = MetadataReaderOptions.Default, MetadataStringDecoder utf8Decoder = null)
+        {
+            var metadata = reader.GetMetadataBlock();
```

Comment:
Nit: also, according to the coding style guidelines, the type should be specified here instead of `var`.